public/react-services/saved-objects.js

```diff
@@ -28,11 +29,30 @@ export class SavedObject {
    */
   static async getListOfIndexPatterns() {
     try {
-      const result = await GenericRequest.request(
+      const savedObjects = await GenericRequest.request(
         'GET',
         `/api/saved_objects/_find?type=index-pattern&fields=title&fields=fields&per_page=9999`
       );
-      return ((result || {}).data || {}).saved_objects || [];
+      let indexPatterns = ((savedObjects || {}).data || {}).saved_objects || [];
+
+      let indexPatternsFields;
+      if(satisfyKibanaVersion('<7.11')){
+        indexPatternsFields = indexPatterns.map(indexPattern => JSON.parse(indexPattern.attributes.fields));
+      }else if(satisfyKibanaVersion('>=7.11')){
+        indexPatternsFields = await Promise.all(indexPatterns.map(async indexPattern => {
+          try{
+            const {data: {fields}} = await GenericRequest.request(
+              'GET',
+              `/api/index_patterns/_fields_for_wildcard?pattern=${indexPattern.attributes.title}`,
+              {}
+            );
+            return fields;
+          }catch(error){
+            return [];
```

Comment:
it could be a false positive, maybe the better is let it throw ?

Reply:
If the index pattern is created, but there are no indices, then this request fails, and as result, the method fails.